.github/workflows/ci.yml

```
@@ -170,24 +170,175 @@ jobs:
             echo "✅ Bun audit completed"
           fi
 
+  # E2E tests with Playwright
+  # Requires GitHub Secret: E2E_DB_PASSWORD for PostgreSQL authentication
+  e2e-tests:
+    name: E2E Tests
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    needs: [lint, test]  # Run after unit/integration tests pass
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)